File: 2942. Find Words Containing Character/findWordsContaining.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution {
    public:
        vector<int> findWordsContaining(vector<string>& words, char x) {
            vector<int> ans;
    
            for(int i=0 ; i<words.size(); i++){
                for(auto chr : words[i]){
                    if(chr == x){
                        ans.push_back(i);
                        break;
                    }
                }
            }
    
            return ans;
        }
    };

File: 2942. Find Words Containing Character/findWordsContaining.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function (words, x) {
    let ans = []

    for (let i = 0; i < words.length; i++) {
        for (let chr of words[i]) {
            if (chr === x) {
                ans.push(i)
                break;
            }
        }
    }

    return ans
};